lib/experimental/block-comments.php

```diff
@@ -93,3 +93,26 @@ function exclude_block_comments_from_admin( $clauses, $query ) {
 	}
 	add_action( 'comments_clauses', 'exclude_block_comments_from_admin', 10, 2 );
 }
+
+/**
+ * Filter the comment count query to exclude block_comment type comments.
+ *
+ * Note: we need to make sure this doesn't interfere with the "Editorial Comments" view
+ * once https://github.com/WordPress/gutenberg/issues/71621 is implemented.
+ *
+ * @param string $query The SQL query string.
+ * @return string The modified SQL query string.
+ */
+if ( ! function_exists( 'filter_comment_count_query_exclude_block_comments' ) ) {
+	function filter_comment_count_query_exclude_block_comments( $query ) {
```

Comment:
Out of curiosity, why do we use this pattern for block comment filters instead of using the `gutenberg_` prefix?

Reply:
I think I saw the pattern in an existing file and followed it. I will fix by adding a gutenberg_ prefix.